### DISCUSSION TITLE
Chainladder in Excel

### DISCUSSION THREAD

**Discussion by mtucker0922** [casact/chainladder-python#543] [General]
Is anyone able to install the chainladder package using the Python in Excel feature? I'm not having luck and guessing it is due to the version of Python that is ran in Excel. Does anyone know if that is the correct reason for the issue or if there is a hope to get this package available to use in Excel in the future?

**Comment by jbogaardt** [comment]
I don't know much about Python in Excel, but I don't think this is possible without coordination from Anaconda. They control the python environment used by Excel including the distribution of packages available. They'd have to include it and ensure compatibility with the rest of the environment.
